### ISSUE TITLE
[Feat Req] Any chance of specifying target graph resolution?

### ISSUE THREAD

**Issue by jaxjexjox** [ansemjo/speedtest-plotter#51]
-e RES_WIDTH 1920
-e RES_HEIGHT 1080

etc?

Would be really nice.

**Comment by ansemjo** [comment]
The output `/result.svg` is a vector graphic anyway, which you can arbitrarily resize (by a factor of 1.5 in this case) – unless you're using an old version which still uses `.png` ..

But sure, setting this line with variables should be easy to do: https://github.com/ansemjo/speedtest-plotter/blob/29ca8b9d512563e50c78126130ad0c9b09b7792b/plotscript#L11

**Comment by jaxjexjox** [comment]
Can I do this via an enviroment variable, should I map plotscript out with -v at launch, or manually edit the file, inside the container each time?

(I'm a B- skill with this, sorry)

**Comment by ansemjo** [comment]
You could edit the file manually for now. I'm not at home right now so I can't change this upstream but I will probably make this an environment variable in the next few days ..

**Comment by jaxjexjox** [comment]
I appreciate your hard work, I'm having difficulty with VI because it's awful, so I might just wait the extra days.

Pretty decent app, this is the 3'rd one I'm on, due to people breaking their work though!

**Comment by ansemjo** [comment]
> ..., due to people breaking their work though!

What do you mean? People just stopping development or what do you mean by "breaking"?

Also, what exactly do you want to achieve with this resolution change? I've just tried some different values and noticed that all the padding and label positions etc. are given relative to the entire plot area; so they don't scale cleanly. Making sure the plot looks good at any range of resolutions would be a little more work than just making the resolution itself a variable ...

For example, `600 x 200`:
![image](https://user-images.githubusercontent.com/11139925/132197572-e032b1a9-db0c-4354-b4c2-ad56f45e7fa8.png)
vs. `6000 x 2000`:
![image](https://user-images.githubusercontent.com/11139925/132197646-a0792727-d76b-45db-b0a4-44a52fc69345.png)

**Comment by ansemjo** [comment]
Implemented fontsize-aware positioning and configurable output resolution. See the [added docs](https://github.com/ansemjo/speedtest-plotter/blob/209f599a41105e698dd675d32dd52ba82b657507/README.md#font-and-resolution) on usage.

tl;dr: add `-e RESOLUTION=1920,1080` to your container and play with the font size `-e FONT=,14` if it becomes too small

**Comment by jaxjexjox** [comment]
I'll need to pull down a new docker image I imagine, right?  
I just spent 5 minutes testing this, realising, I forgot to pull a new image.

Thank you!

**Comment by ansemjo** [comment]
Yes and you also need to wait for [the automated build workflow](https://github.com/ansemjo/speedtest-plotter/actions/runs/1215169934) to finish first! :)
